lib/desired-caps.js

```diff
@@ -33,6 +33,9 @@ let desiredCapConstraints = _.defaults({
   tapWithShortPressDuration: {
     isNumber: true
   },
+  scaleFactor: {
+    isString: true
```

Comment:
Should this be a number? What are the valid values of the `scaleFactor`? If it is closely circumscribed the validation might need to be done manually, in [validateDesiredCaps](https://github.com/appium/appium-xcuitest-driver/blob/master/lib/driver.js#L534)